.eslintrc.json

```diff
@@ -1,5 +1,11 @@
 {
   "root": true,
+  "plugins": [
+    "eslint-plugin-no-jquery"
+  ],
+  "extends": [
+    "plugin:no-jquery/deprecated"
+  ],
```

Comment:
In rules below you will also need to add:
```
    "no-jquery/no-event-shorthand": "error",
```

Reply:
My bad. I thought this rule was not included in the "plugin:no-jquery/deprecated". Turns out we just had two extends keys :)

CI is failing now with my latest edit to rectify this.